SCClassLibrary/Common/Core/Color.sc

```diff
@@ -109,6 +109,31 @@ Color {
 		^Color.hsv(b[0], b[1], blend(f[2], b[2], blend), alpha)
 	}
 
+	*ryb { arg red, yellow, blue, alpha=1;
+		var basis = #[
+			[1, 1, 1], // white
+			[1, 0, 0], // red
+			[1, 1, 0], // yellow:
+			[0.163, 0.373, 0.6], // blue
+			[0.5, 0, 0.5], // violet
```

Comment:
in the paper you linked, this is [0.5, 0.5, 0.0]

Reply:
although if i understand correctly it makes more sense as what you've written, since the first and third indices are red and blue respectively